website/static/css/style.css

```diff
@@ -666,4 +666,8 @@ button.close {
 
 .comma-separated:last-of-type:after {
     content: "";
-}
\ No newline at end of file
+}
+
+#render-node .ball-pulse-sync>div, #render-node .ball-pulse>div, #render-node .ball-scale>div, #render-node .ball-beat-sync>div, #render-node .ball-beat>div, #render-node .ball-beat>div {
```

Comment:
The new loader is only in my-projects.css.  I would think we would want to be able to reference it everywhere.  Am I missing something @sloria?